src/test/fuzz/process_message.cpp

```diff
@@ -75,6 +75,10 @@ void fuzz_target(const std::vector<uint8_t>& buffer, const std::string& LIMIT_TO
                                                 GetTime<std::chrono::microseconds>(), std::atomic<bool>{false});
     } catch (const std::ios_base::failure&) {
     }
+    {
```

Comment:
Do I understand correctly that we do not need `try { ... } catch (const std::ios_base::failure&) {}` here because the inputs to `SendMessages()` have effectively been filtered for "sane inputs" by `ProcessMessage()`?

Reply:
std::ios_base::failure is thrown by the serialization framework when deserialization fails.  SendMessages AFAIK doesn't deserialize anything, it just acts on already parsed data. So I wouldn't say it's just been "filtered" - it has been completely processed already, and the result of that processing may trigger responses in SendMessages.

Reply:
That makes sense. Thanks, @sipa !